ykrt/try_passes.py

```diff
@@ -69,11 +64,14 @@ def get_all_passes(flagType):
     assert(p.returncode == 0)
 
     sout = sout.decode()
-    pass_descrs = list(set([x.strip() for x in sout.strip().split("\n")]))
-    
+    # order is getting messed up because of using set, since set is
```

Comment:
I think this comment describes the past state of the code a little more than necessary.

It could either be removed or rephrased to something like:

> We are careful to maintain the ordering of the passes.

Or similar.

Reply:
removed the comment.